docs/user-guide/masking.ipynb

```diff
@@ -112,6 +112,14 @@
     "sc.show(a)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the example above the mask was specified as the data array was created.\n",
+    "But like for `coords` masks can also be assigned after the data array was created:"
```

Comment:
[Minor]
```suggestion
    "However, similar to `coords`, `masks` can also be assigned on an existing `DataArray` instance:"
```
Sth like this maybe...?

Reply:
I am not sure we can assign to `masks`? I suppose you mean setting items?

Reply:
`assign_masks` return a new object, yes. Also, the example right after this text sets a mask. It is not assigning the `masks` property.